src/controllers/bookmarkController.js

```diff
@@ -1,17 +1,18 @@
 import bookmarkService from "../services/bookmarkService.js";
+import { ContentTypeMismatchError } from "../utils/customError.js";
 
-const getAllBookmarks = async (req, res) => {
+const getAllBookmarks = async (req, res, next) => {
```

Comment:
Nitpick: While all the controllers follow the same pattern, this could be made simpler with the help of a library.

Reply:
Thanks for the suggestion this is done as well